Sources/Component/Label/LabelStyle.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import UIKit
+
+/// A data structure describing the styles of a `Label`.
+public struct LabelStyle {
+    /// The color used to render the text of a `Label`.
+    public let color: UIColor
+
+    /// The font used to render the text of a `Label`.
+    public let font: UIFont
+
+    /// The number of lines the `Label` is allowed to line break.
+    public let numberOfLines: Int
```

Comment:
If you provide the numberOfLines you should also give the possiblity to adjust lineBreakMode and textAlignment for completeness. Usually these three properties are used together.